scarb/src/ops/new.rs

```diff
@@ -193,6 +199,19 @@ fn mk(
         "#})
     }
 
+    if snforge {
+        init_snforge(name)?;
```

Comment:
I am not sure if we want to both create the package here with Scarb and then with Snforge. Will Snforge overwrite what has been created by Scarb? It does call Scarb `new` once again under the hood, is it ok?

Reply:
Changed forge behaviour in https://github.com/foundry-rs/starknet-foundry/pull/2113